impeller/aiks/paint.cc

```diff
@@ -107,4 +107,8 @@ std::shared_ptr<FilterContents> Paint::MaskBlurDescriptor::CreateMaskBlur(
                                             effect_transform);
 }
 
+bool Paint::HasFilters() const {
+  return color_filter.has_value();
```

Comment:
Why color filters specifically? Not all color filters absorb opacity IIRC.

Reply:
Good point, I should check for this with https://github.com/flutter/engine/pull/39266

The general answer is that the high level framework API makes it extrenely easy to apply color blend filters to images, and no other type of color or image filter. see color and colorBlendMode in https://api.flutter.dev/flutter/widgets/Image-class.html

Reply:
As of now, all color filters absorb opacity.